docs/simulation_models.rst

```diff
@@ -2,4 +2,205 @@
+
+.. math::
+    Re = \frac{Dv\rho}{\mu}.
+
+:math:`\mu` is the dynamic viscosity of water.
+
+In a pipe, flow is laminar if :math:`Re` < 2300 and turbulent if :math:`Re` > 4000.
+In district heating pipes, flow is usually turbulent. The turbulent flow regime can be further
+distinguished into smooth, intermediate and rough regime depending on the pipe's surface roughness.
+
+In Bordin2015 :math:`\lambda` is calculated using the approximation formula:
+
+.. math::
+    \lambda = 0.07 \cdot Re ^{-0.13} \cdot D^{-0.14}.
+
+A more accurate approximation of the Colebrook-White-equation for flow in pipes is given by this
+formula:
+
+.. math::
+    \lambda = \frac{1.325}{(ln(\frac{\epsilon}{3.7D} + \frac{5.74}{Re^{0.9}}))^2}.
+
+**Local pressure losses**
+
+Local pressure losses are losses at junction elements, angles, valves etc. They are described by
+the localized pressure loss coefficient :math:`\zeta`:
+
+.. math::
+    \Delta p_{loc} = \zeta \frac{v^2}{2} \rho
+It is assumed that each fork has a tee and each consumer has a valve installed.
+
+**Hydrostatic pressure difference**
+
+The hydrostatic pressure difference is calculated as follows:
+.. math::
+    \Delta p_{hydrostatic} = - \rho g \Delta h
+
+
+**Pump power**
+
+Mass flow in the pipes is driven by a pressure difference that is generated by the pumps.
+They maintain the pressure losses inside the pipes. Pump power thus depends on the pressure losses
+:math:`\Delta p`, the mass flow :math:`\dot{m}` and the efficiency
+:math:`\eta_{pump} = \eta_{el} \cdot \eta_{hyd}` of the pumps.
+
+.. math::
+    P_{el. pump} = \frac{1}{\eta_{el}\eta_{hyd}}\frac{\Delta p }{\rho} \dot{m}
+
+
+Thermal equations
+~~~~~~~~~~~~~~~~~
+
+The temperature spread defines the amount of heat that is transported with a given mass flow:
+
+.. math::
+    \dot{Q} = \dot{m} \cdot c \cdot \Delta T.
+
+
+A larger temperature spread allows smaller pipe's diameters, which reduces the
+investment cost of new pipes or increases the thermal power of existing pipes.
+
+**Heat losses**
+
+Heat losses depend on temperature level, mass flow and pipe insulation.
+Especially the representation of the heat losses depends a lot on the level of detail of a model.
+As mentioned above, the current implementation treats the thermal beaviour as steady state.
+The temperature at the outlet is calculated as follows:
+
+.. math::
+    T_{out} = T_{env} + (T_{in} - T_{env}) \cdot exp\{-\frac{U \pi D L}{c \cdot \dot{m}}\}.
+
+
+Where :math:`T_{in}` and :math:`T_{out}` are the temperatures at the start and end of the pipe,
+:math:`T_{env}` the environmental temperature and :math:`U` the thermal transmittance.
+
+.. math::
+    U_{spez} = U \cdot \pi D &\text{\hspace{1cm} for single pipes}\\
+    U_{spez} = U \cdot 2 \pi D &\text{\hspace{1cm} for double pipes}
+
+In data documentation of district heating pipes, you ofter find the value for the specific heat
+loss per meter :math:`U_{spez} [W/(K m)]`.
+
+
+References
+----------
```

Comment:
```suggestion
----------

.. [1] Chiara Bordin. Mathematical Optimization Applied to Thermal and Electrical Energy Systems. PhD
    thesis, Università di Bologna, 2015.
```

Further references such as the one for Colebrook-White-equation (see above) should be gathered here. Which ones are used in the docs?

Reply:
You can use citations in sphinx as shown here: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#citations

For example: In [Bordin2015]_, ...

And in the Bibiliography you include

[Bordin2015] Author, Title, etc.